sdk/communication/azure-communication-administration/README.md

```diff
@@ -13,7 +13,7 @@ Acquired phone numbers can come with many capabilities, depending on the country
 - An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
 - [Java Development Kit (JDK)](https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable) version 8 or above.
 - [Apache Maven](https://maven.apache.org/download.cgi).
-- A deployed Communication Services resource.
+- A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/module/az.communication/new-azcommunicationservice?view=azps-5.3.0) to set it up.
```

Comment:
Let's not specify the azure powershell version in the link:  https://docs.microsoft.com/en-us/powershell/module/az.communication/new-azcommunicationservice

Reply:
Excellent point, changing that